README.md

```diff
@@ -0,0 +1,27 @@
+# App
```

Comment:
It'd be worth updating this file so it has a brief description of the project, instead of the default stuff